.github/ISSUE_TEMPLATE/proposal--featured-talk.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Talk proposal - Feature Talk (max 25 minutes)
+about: Propose a feature talk. Max 25 minutes in length
+title: "Talk proposal: <talk title>"
+labels: Feature
+assignees: ""
+---
+
+# Title of talk
+
+A short description of the talk. The goal should be to explain to people WHY this topic is interesting and HOW it will help them learn decentralized web protocols and tools. 1-2 paragraphs is fine.
+
+Include a short bio--anything that gives us an idea of why you're interested to speak on this subject. Feel free to include any relevant links.
+
+- Name
+- [Twitter]()
+- [Github]()
+- [Website]()
+
+* [ ] I've read and agree to the [ProtoSchool Seattle Code of Conduct](https://github.com/jimcal/seattle/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
```suggestion
* [ ] I've read and agree to the [ProtoSchool Seattle Code of Conduct](https://github.com/jimcal/seattle/blob/master/CODE_OF_CONDUCT.md)
* [ ] I'm new to giving talks and want to practice present to an organizer over a video call first
```

Reply:
(just an idea)

Reply:
I like it! I don't have the mentoring experience for new speakers (yet). Is this something we want to committing to support? I am excited about the idea, at the same time I want to have a healthy discussion even if the next three months no one request this.

Reply:
it's a good question. I think I'd be able to hold that role for at least the first few/don't imagine it as a big timesuck but hard to really know

Reply:
Thanks for willing to help! I will add this into the revision commit I am making.